src/main/java/com/klarna/hiverunner/StandaloneHiveRunner.java

```diff
@@ -158,7 +159,12 @@ protected final void runTestMethod(FrameworkMethod method,
     public HiveShellContainer evaluateStatement(List<? extends Script> scripts, Object target,
         Path temporaryFolder, Statement base) throws Throwable {
         container = null;
-        FileUtil.setPermission(temporaryFolder.toFile(), FsPermission.getDirDefault());
+        File temporaryFile = temporaryFolder.toFile();
+        if (Files.notExists(temporaryFolder)) {
+            temporaryFile = new File(temporaryFolder.toString());
```

Comment:
I don't think we need this line, it's just creating a `File` object, we should be able to use `temporaryFile` as-is.

Reply:
True!